src/main/java/org/example/algorithms/sorting/TimeSort.java

```diff
@@ -0,0 +1,124 @@
+package org.example.algorithms.sorting;
```

Comment:
It seems like this is copied from GFG https://www.geeksforgeeks.org/timsort/. So I am marking this PR invalid.